helm/values.yaml

```diff
@@ -0,0 +1,142 @@
+# Default values for amazon-cloudwatch-observability.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+## Provide a name in place of cloudwatchagent-operator.
+##
+nameOverride: ""
+
+## Reference one or more secrets to be used when pulling images from authenticated repositories.
+imagePullSecrets: [ ]
+
+## Provide the ClusterName (optional when installing via EKS add-on)
+clusterName: EKS_CLUSTER_NAME
+
+## Provide the Region (optional when installing via EKS add-on)
+region: AWS_REGION_NAME
+
+containerLogs:
+  enabled: true
+  fluentBit:
+    image:
+      repository: public.ecr.aws/aws-observability/aws-for-fluent-bit
+      tag: 2.31.12.20230911
+
+## Provide CloudWatchAgent Operator manager container image and resources.
+##
+manager:
+  name:
+  image:
+    # This repository is for internal beta testing only, it will be replaced with the public repository before public release
+    repository: 506463145083.dkr.ecr.us-west-2.amazonaws.com/cwagent-operator-release
```

Comment:
can we change these to use the public ecr repo?

Reply:
Yup let me update these now